ocaml/nbd/src/main.ml

```diff
@@ -176,15 +176,7 @@ let main port certfile _ =
       (ignore_exn_delayed (fun () -> Lwt_unix.close sock))
   in
   (* Log unexpected exceptions *)
-  let () =
-    Lwt_main.run
-      (Lwt.catch t (fun e ->
-           Lwt_log.fatal_f "Caught unexpected exception: %s"
-             (Printexc.to_string e)
-           >>= fun () -> Lwt.fail e
```

Comment:
The exception is no longer reraised, instead it's ignored. I think in case of unhandled exception nbd should fail with an error code.

I recommend changing this back to how it was and then change `fatal_f` for `error_f`